Composer/packages/client/src/plugins/api.ts

```diff
@@ -35,7 +35,7 @@ interface PublishAPI {
   getName?: () => string;
   savePublishConfig?: (config: PublishConfig) => void;
   getTokenFromCache?: () => { accessToken: string; graphToken: string };
-  isGetTokenFromUser?: () => boolean;
+  userShouldProvideTokens?: () => boolean;
```

Comment:
@benbrown mentioned that other extensions outside Composer exist (I'm not sure). I don't know if this name change would mean a breaking contract change for them.

Reply:
This is a good point. What I'll do is add the old method back, but tag it as deprecated.